src/App.tsx

```diff
@@ -1,60 +1,95 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
 import classNames from 'classnames';
-import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
-import { Loader } from './components/Loader';
+import { getUsers } from './utils/fetchClient';
+import { User } from './types/User';
+import { PostsList } from './components/PostsList';
+import { Post } from './types/Post';
 
 export const App: React.FC = () => {
+  const [showSideBar, setShowSideBar] = useState(false);
+  const [users, setUsers] = useState<User[] | null>();
```

Comment:
Set the initial value to the user variable
```suggestion
  const [users, setUsers] = useState<User[] | null>(null);
```

Reply:
Done